.github/workflows/failureNotifier.yml

```
@@ -0,0 +1,85 @@
+name: Notify on Workflow Failure
+
+on:
+  workflow_run:
+    workflows: ["Process new code merged to main"]
+    types:
+      - completed
+
+permissions:
+  issues: write
+
+jobs:
+  notify-failure:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      - name: Fetch Workflow Run Jobs
+        id: fetch-workflow-jobs
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const runId = "${{ github.event.workflow_run.id }}";
+            const jobsData = await github.rest.actions.listJobsForWorkflowRun({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: runId,
+            });
+            return jobsData.data;
+
+      - name: Process Each Failed Job
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const jobs = ${{steps.fetch-workflow-jobs.outputs.result}};
```

```suggestion
            const jobs = ${{ steps.fetch-workflow-jobs.outputs.result }};
```
Sorry I realised that this can be left with spacing :smile: